pkg/controller/toolchainstatus/toolchainstatus_controller_test.go

```diff
@@ -1101,19 +1162,25 @@ func newDeploymentWithConditions(deploymentName string, deploymentConditions ...
 	}
 }
 
-func newGetMemberClustersFuncReady(fakeClient client.Client) cluster.GetMemberClustersFunc {
-	return func(conditions ...cluster.Condition) []*cluster.CachedToolchainCluster {
-		return []*cluster.CachedToolchainCluster{
-			cachedToolchainCluster(fakeClient, "member-1", corev1.ConditionTrue, metav1.Now()),
-			cachedToolchainCluster(fakeClient, "member-2", corev1.ConditionTrue, metav1.Now()),
+type getMemberClustersFunc func(client.Client) cluster.GetMemberClustersFunc
+
+func memberClusters(clusterNames ...string) getMemberClustersFunc {
+	return func(cl client.Client) cluster.GetMemberClustersFunc {
+		return func(conditions ...cluster.Condition) []*cluster.CachedToolchainCluster {
+			clusters := make([]*cluster.CachedToolchainCluster, len(clusterNames))
+			for i, clusterName := range clusterNames {
+				clusters[i] = cachedToolchainCluster(cl, clusterName, corev1.ConditionTrue, metav1.Now())
+			}
+			return clusters
 		}
 	}
 }
 
-func newGetMemberClustersFuncEmpty(_ client.Client) cluster.GetMemberClustersFunc {
-	return func(conditions ...cluster.Condition) []*cluster.CachedToolchainCluster {
-		var clusters []*cluster.CachedToolchainCluster
-		return clusters
+func noMemberClusters() getMemberClustersFunc {
```

Comment:
do you really need this function - couldn't you just use `memberClusters()` as you did here: https://github.com/codeready-toolchain/host-operator/blob/761645b91eeeb0ea513c2879734fe85cea63e22e/pkg/controller/toolchainstatus/toolchainstatus_controller_test.go#L433

Reply:
you're right. I thought that this `noMemberCluster()` func would be more expressive than something like `memberClusters()` (and I did not realize I used it elsewhere).
That being said, I'm fine with dropping `noMemberClusters()` and using `memberClusters()` instead. 
Let me know what you prefer.

Reply:
I prefer having just `memberClusters()` and dropping `noMemberCluster()`

Reply:
ok, sounds good to me! 
done in 2c92725